GVRf/Framework/src/org/gearvrf/GVRViewManager.java

```diff
@@ -462,6 +463,11 @@ void onDrawEyeView(int eye, float fovDegrees) {
         if (!(mSensoredScene == null || !mMainScene.equals(mSensoredScene))) {
             GVRCameraRig mainCameraRig = mMainScene.getMainCameraRig();
 
+            if(eye == 0) {
+                GVRPerspectiveCamera centerCamera = mainCameraRig.getCenterCamera();
+                cull(mMainScene.getNative(), centerCamera.getNative(), mRenderBundle.getMaterialShaderManager().getNative());
+            }
+
```

Comment:
OK. Maybe by the end of this week to see if you can handle this. After that if there is still a problem, we can merge it firstly and then figure out if we can move it to onDrawFrame with another pull request.